bank_component.rb

```diff
@@ -0,0 +1,97 @@
+=begin
+ Feature 1: 
+ This feature will allow user to either sign up for an account.
+ Or log in with their last name,last 4 digits of their social security number, and password.        
+=end
+
+
+
+#class for log in and sign up method
+class LogInSignUp
+
+    @@user_info=[
+        {first:'andy', last:'park', password:'hello123', ssn:'123456789'},
+        {first:'Jason', last:'kim', password:'bye123', ssn:'135791098'}
+    ]
+
+    #Instance variable to determine if the user's credential is validated
+    @user_credential_validation = false
+    
+    attr_accessor :last_name, :password, :ssn
+
+    def initialize(last_name, password, ssn)
+        @last_name = last_name
+        @password=password
+        @ssn=ssn
+    end
+
+    def validate_log_in_credentials
+        @@user_info.each do |x|
+            if x[:last].to_s == @last_name && x[:password] == @password && x[:ssn][5,8] == @ssn
+                puts "Welcome back #{x[:first]} #{x[:last]}!"
+                return
```

Comment:
Ah, I see that this method is supposed to return once a match is found. 
There is a handy ruby method that could help shorten this whole method a bit and avoid having to explicitly `return`.

Check out Enumerable#find
https://apidock.com/ruby/Enumerable/find

P.S. Avoiding explicit returns is not a style requirement, but it generally means that things could be cleaner.